src/privileged/vpnRecommender/content/doorhanger/doorhanger.html

```diff
@@ -0,0 +1,57 @@
+<!-- * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!DOCTYPE HTML>
+
+<html>
+  
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <link  rel="stylesheet" type="text/css" href="./doorhanger.css" />
+  </head>
+
+
+  <body>
+      <div id="recomcontainer">
+
+        <div id="recomcontainer-top">
+          <div id="headersection">
+            <div id="left-info" class="info">
+            </div>
+            <div id="headercontainer">
+              <div id="header">
+                Recommendation
+              </div>
+            </div>
+            <div id="right-info" class="info">
```

Comment:
Just noting that I found this icon extremely confusing, I thought I could click it to get more info about this "Recommendation".

Reply:
This will indeed send the user to the support page. The page is not ready yet, hence the confusion. Eventually, we will add it in.